src/opcodes.rs

```diff
@@ -890,6 +890,10 @@ lazy_static! {
             .with_rmw()
             .into_table(&mut ops);
         // Begin cmp opcodes
+        //0x0F C7 CMPXCHG8B  
+        define_opcode(0xC7).is_two_byte_op().calls(cmpxchg8b).with_gas(Low)
+            .with_arg(ArgSource::ModRM, OpcodeValueSize::Fixed(ValueSize::Dword))
```

Comment:
Shouldn't this be ValueSize::Qword?

Reply:
No. This is to get the address, not the Qword itself. The address points to the Qword.